test/e2e/core/platform_installer_e2e.test.mjs

```diff
@@ -196,7 +183,9 @@ describe('PackageInstallerE2E', () => {
   describe('copyPlatformConfigFiles', () => {
     it('should succeed to copy platform config files for node0', async () => {
       const podName = 'network-node0-0'
-      await installer.resetHapiDirectories(podName)
+      await k8.execContainer(podName, constants.ROOT_CONTAINER, ['bash', '-c', `rm -f ${constants.HEDERA_HAPI_PATH}/*.txt`])
+      await k8.execContainer(podName, constants.ROOT_CONTAINER, ['bash', '-c', `rm -f ${constants.HEDERA_HAPI_PATH}/*.xml`])
+      await k8.execContainer(podName, constants.ROOT_CONTAINER, ['bash', '-c', `rm -f ${constants.HEDERA_HAPI_PATH}/data/config/*.properties`])
```

Comment:
This and the commands above will fail once I finish the configMap and secretMapping:
https://github.com/hashgraph/full-stack-testing/pull/797/files

configMap:
network-node-data-config-cm: /opt/hgcapp/services-hedera/HapiApp2.0/data/config
	• api-permission.properties
	• application.properties
	• bootstrap.properties
	• https://github.com/hashgraph/solo/issues/136
network-node-etc-cm: /etc/network-node
	• application.env
	• https://github.com/hashgraph/solo/issues/110
network-node-hapi-app-cm: /opt/hgcapp/services-hedera/HapiApp2.0
	• config.txt
	• settings.txt
	• log4j2.xml
	• settingsUsed.txt
	• https://github.com/hashgraph/solo/issues/136
network-node-platform-cm: /home/hedera/platform
	• build-${tag}.zip
	• build-${tag}.sha384
	• https://github.com/hashgraph/solo/issues/89

secrets:
network-node-hapi-app-secrets: /opt/hgcapp/services-hedera/HapiApp2.0/
	• hedera.crt
	• hedera.key
	• https://github.com/hashgraph/solo/issues/135
network-node-keys-secrets: /opt/hgcapp/services-hedera/HapiApp2.0/data/keys
	• a-private-node0.pem
	• a-public-node0.pem
	• a-public-node1.pem
	• a-public-node2.pem
	• s-private-node0.pem
	• s-public-node0.pem
	• s-public-node1.pem
	• s-public-node2.pem
	• https://github.com/hashgraph/solo/issues/135